### DISCUSSION TITLE
i want to make an application which make changes inside doc files using budibase

### DISCUSSION THREAD

**Discussion by AryanGaur14** [Budibase/budibase#13312] [Help]
I'm seeking assistance in creating a project that modifies content within DOC or DOCX files. These files contain predefined templates. My objective is to collect user input, store it in a database, display the data to users, and upon user selection, present a list of files. Upon choosing a file from the list, I aim to trigger changes within the downloaded DOC file template. I intend to accomplish this using open-source, no-code platforms like Budibase. However, I'm facing a challenge as i don't know how to use it with zapier or make .

Please note that I'm entirely new to no-code platforms and wish to achieve this automation without coding. The workflow I envision is as follows: a form collects user-entered details, stores them in a database, displays the data to users, and upon selection, presents a list of files. If any of these files are clicked, the chosen file should be downloaded and returned to the page.

**Comment by ConorWebb96** [comment]
Hey @AryanGaur14,

We are not a `no code` platform. We are a `low code` platform. 

I don't believe this is something which is currently possible with Budibase i'd recommend creating a feature request if you feel this sort of functionality should be added to Budibase.
